<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="spinBox"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_2"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="listView1"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButtonAdd">
         <property name="text">
          <string>+</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSubstr">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonMult">
         <property name="text">
          <string>*</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDiv">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtoCLS">
         <property name="text">
          <string>CLS</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu1">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAction1"/>
   </widget>
   <addaction name="menuMenu1"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAction1">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
